src/modules/drawing/lib/drawingUtils.js

```diff
@@ -15,14 +15,15 @@ export function formatPointCoordinates(coo) {
     return `${format(coo[0], 0)}, ${format(coo[1], 0)}`
 }
 
-export function formatMeters(value, { dim = 1, digits = 2 } = {}) {
+export function formatMeters(value, { dim = 1, digits = 2, applyFormat = true } = {}) {
     const factor = Math.pow(1000, dim)
     let unit = dim === 1 ? 'm' : 'm²'
     if (value >= factor) {
         unit = dim === 1 ? 'km' : 'km²'
         value /= factor
     }
-    return `${format(value, digits)} ${unit}`
+    value = applyFormat ? format(value, digits) : value.toFixed(digits)
```

Comment:
What about calling this function with `digits=0` instead of adding a new boolean arg? It will have the same output I think

Reply:
No, it is not the same. `digits` defines the number of digits after the dot, `applyFormat=false` disables inserting of " ' " for thousands